openmdao/api.py

```diff
@@ -0,0 +1,58 @@
+#components
+from openmdao.components.constraint import ConstraintComp
+from openmdao.components.exec_comp import ExecComp
+from openmdao.components.external_code import ExternalCode
+from openmdao.components.linear_system import LinearSystem
+from openmdao.components.meta_model import MetaModel
+from openmdao.components.multifi_meta_model import MultiFiMetaModel
+from openmdao.components.indep_var_comp import IndepVarComp
+from openmdao.components.param_comp import ParamComp  #deprecated
+from openmdao.components.unit_comp import UnitComp
+#core
+from openmdao.core.component import Component
+from openmdao.core.group import Group
+from openmdao.core.parallel_group import ParallelGroup
+from openmdao.core.problem import Problem
+from openmdao.core.system import System
+from openmdao.core.driver import Driver
+from openmdao.core.basic_impl import BasicImpl
+from openmdao.core.vec_wrapper import VecWrapper, SrcVecWrapper, TgtVecWrapper
+from openmdao.core.relevance import Relevance
+#drivers
+from openmdao.drivers.scipy_optimizer import ScipyOptimizer
+try:
+    from openmdao.drivers.pyoptsparse_driver import pyOptSparseDriver
+except ImportError:
+    pass
+#recorders
+from openmdao.recorders.base_recorder import BaseRecorder
+from openmdao.recorders.dump_recorder import DumpRecorder
+from openmdao.recorders.shelve_recorder import ShelveRecorder
+from openmdao.recorders.sqlite_recorder import SqliteRecorder
+#solvers
+from openmdao.solvers.ln_direct import DirectSolver
+from openmdao.solvers.ln_gauss_seidel import LinearGaussSeidel
+from openmdao.solvers.newton import Newton
+from openmdao.solvers.nl_gauss_seidel import NLGaussSeidel
+from openmdao.solvers.run_once import RunOnce
+from openmdao.solvers.scipy_gmres import ScipyGMRES
+from openmdao.solvers.solver_base import LinearSolver, NonLinearSolver
+try:
+    from openmdao.solvers.petsc_ksp import PetscKSP
+except ImportError:
+    pass
+#surrogate models
+from openmdao.surrogate_models.kriging import KrigingSurrogate, FloatKrigingSurrogate
+from openmdao.surrogate_models.multifi_cokriging import MultiFiCoKrigingSurrogate, \
+    FloatMultiFiCoKrigingSurrogate
+from openmdao.surrogate_models.nearest_neighbor import NearestNeighbor
+from openmdao.surrogate_models.response_surface import ResponseSurface
+from openmdao.surrogate_models.surrogate_model import SurrogateModel, \
+    MultiFiSurrogateModel
+#units
+from openmdao.units.units import PhysicalQuantity, NumberDict, UnitsOnlyPQ, PhysicalUnit, \

```

Comment:
`get_conversion_tuple` and `convert_units` are the only methods we publicly declare in the units package